### PR TITLE
Minor changes to avoid flake8 errors

### PR DESCRIPTION
**Pull request by hailihu**

### CODE REVIEW

File: tests/test_document.py [hunk 1]
```diff
@@ -27,17 +26,14 @@ def test_create(self):
         self.assertEqual(doc.id, test_other_id)
         self.assertEqual(doc.value, {'_id': test_other_id})
 
-
-    def test_no_id(self):
-        doc = Document({'someattr': 1})
-        self.assertRaises(AttributeError, getattr(doc), 'id')
-        self.assertRaises(AttributeError, getattr(doc), 'rev')
-
+    # def test_no_id(self):
```
Comment (lnauta) [inline tests/test_document.py:29]: Why is this commented out?

**Comment by lnauta** [comment]
Is there a fundamental break by renaming executers.py or just a small mistake somewhere?

**Comment by lnauta** [comment]
I just realized the executors rename is in a feature branch, thats why this merge needs the old name.

**Comment by lnauta** [comment]
Looks good, once the conversation is resolved it can be merged.

File: tests/test_document.py (hunk 1 above)
Comment (hailihu) [reply tests/test_document.py:29]: It gives error with flake8:
`F811 redefinition of unused 'test_no_id' from line 29`
I.e. the function id defined twice and unused.

Comment (lnauta) [reply tests/test_document.py:29]: Can you rename the second test_no_id to something like test_id_len or what fits, it does not test a "no id" situation but whether a non-zero id is created (or more precisely, len 10 or longer).

Comment (lnauta) [reply tests/test_document.py:29]: FYI: all methods starting with `test_` are called by pytest, so this "unused" is a bit misleading, the method is not called in other places explicitly

Comment (hailihu) [reply tests/test_document.py]: Thanks, done

Comment (hailihu) [reply tests/test_document.py]: Think I fixed it, please check.

Comment (lnauta) [reply tests/test_document.py]: Looks good!

**Pull request merged**